schema/ingestion_config/migrations/issue_997.py

```diff
@@ -0,0 +1,185 @@
+"""
+This script migrates the ingest config files to a new schema based on the changes outlines in
+https://github.com/chanzuckerberg/cryoet-data-portal/issues/997
+"""
+import glob
+import itertools
+import json
+import traceback
+from typing import Union
+
+import numpy as np
+import yaml
+
+
+def rawtilts_to_collection_metadata(config: dict) -> None:
+    list_globs = []
+    if 'rawtilts' in config:
+        for i in config['rawtilts']:
+            if "sources" not in i:
+                continue
+            old_source = i["sources"][0]["source_multi_glob"]["list_globs"]
+            list_globs.extend(s for s in old_source if s.endswith('.mdoc') or "{mdoc_name}" in s)
+            for source in list_globs:
+                old_source.remove(source)
+        if list_globs:
+            if 'collection_metadata' not in config:
+                config['collection_metadata'] = [{"sources": [{"source_multi_glob": {"list_globs": []}}]}]
+            config["collection_metadata"][0]["sources"][0]["source_multi_glob"]["list_globs"].extend(list_globs)
+
+
+def rawtilts_to_alignments(config: dict) -> None:
+    list_globs = []
+    format_dict = {
+        "IMOD": [],
+        "ARETOMO3": [],
+    }
+
+    def valid_file(file):
+        return any(file.endswith(ext) for ext in ['.tlt', ".xf", ".aln", ".com", ".txt", ".csv"])
```

Comment:
I believe this might not be comprehensive. We might also have to include `".tltx"`. ref: [here](https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/262/files#diff-83deea45e3ed229323bb17084c0392453fa6d839f85cb9f396d977d33940f0d3R357)

Reply:
I'm not finding any file with that suffix in the existing configs.

Reply:
should `.tiltx` be `IMOD` or `ARETOMO3`?

Reply:
do you mean `.xtlt`

Reply:
here are all the supported formats
```
    IMOD_ext = ['.tlt', ".xf", "tilt.com", "news.com", ".xtilt"]
    AreTomo3_ext = ['.aln', ".txt", ".csv"]
```